ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -288,7 +287,7 @@ public ReplicationManager(ReplicationConfig replicationConfig, SSLConfig sslConf
       this.notification = requestNotification;
       this.metricRegistry = metricRegistry;
       this.dataNodeRemoteReplicaInfosPerDC = new HashMap<String, DataNodeRemoteReplicaInfos>();
-      this.sslEnabledDatacenters = Utils.splitString(sslConfig.sslEnabledDatacenters, ",");
+      this.sslEnabledDatacenters = Utils.splitString(clusterMapConfig.clusterMapSslEnabledDatacenters, ",");
```

Comment:
those will be taken as DCname (which is considered to have a whitespace). So, may not work as expected